cli/tests/test_package.py

```diff
@@ -0,0 +1,47 @@
+import os
+import pathlib
+
+import pytest
+import requests
+from click.testing import CliRunner
+
+from functionary.config import save_config_value
+from functionary.package import get_tar_path, publish
+
+
+def response_200(*args, **kwargs):
+    response = requests.Response()
+    response.status_code = 200
+    response._content = b'{"token": "somegreattoken"}'
+
+    return response
+
+
+# Calling publish with --keep ::: Resulting in tarball in the desired location
```

Comment:
These comments should be docstrings for the test function themselves.  So you'd remove this and then do:

```python
@pytest.mark.usefixtures("config")
def test_publish_with_keep(fakefs, monkeypatch):
    """Calling publish with --keep results in tarball not being removed"""
```

That kind of thing.

Reply:
fixed